dev-requirements.txt

```
@@ -6,3 +6,4 @@ memray == 1.6.0; platform_system != "Windows" and sys_platform != "darwin" and p
 pre-commit == 2.20.0
 pytest == 7.2.0
 pytest-cov == 4.0.0
+pytest-benchmark == 4.0.0
```

```suggestion
pytest-benchmark == 4.0.0
pytest-cov == 4.0.0
```
I think putting these in lexicographic order should fix the linter...